Rubberduck.Parsing/ParserRuleContextExtensions.cs

```diff
@@ -164,6 +164,111 @@ private static TContext GetAncestor_Recursive<TContext>(ParserRuleContext contex
             return listener.Matches;
         }
 
+        public static IEnumerable<T> GetChildren<T>(this RuleContext context)
```

Comment:
this should already exist...

Reply:
Removing method - unused in current PR.  It was a remnant of the retracted version.

Reply:
Method is unused - and has been removed.